### PR TITLE
Rf  produce api test

### LINKED ISSUE #43: [REFACTOR] Add linting to produce API
**Is your feature request related to a problem? Please describe.**
Add linting and make necessary file adjustments

**Describe the solution you'd like**
Use PyLint

**Additional context**
branch: rf-produce-api-lint

### PR DESCRIPTION
**Pull request by LadPaule**
## Description
Added tests and requirements file

Closes #43 

## How Has This Been Tested?
Using Pylint

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/services/produce_api/produceAPI/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+argcomplete==1.11.1
```
Comment (kmwamasali) [inline src/services/produce_api/produceAPI/requirements.txt]: change this file to lis only 

Django
django-rest-swagger
djangorestframework
Pillow

**Review by kmwamasali — CHANGES_REQUESTED** [review]
change requirement.txt to list only major dependencies

File: src/services/produce_api/produceAPI/produce/views.py [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
 from .models import Produce
 from .serializers import ProduceSerialiser
 
-
+# creating the create and list products class
 class Product(generics.ListCreateAPIView):
     serializer_class = ProduceSerialiser
     queryset = Produce.objects.all()
 
+# defining a get function, this should return a list of products in the db
```
Comment (amkayondo) [inline src/services/produce_api/produceAPI/produce/views.py:11]: No need for inline comments as your function name can be used to define what the function does.

**Review by amkayondo — CHANGES_REQUESTED** [review]
Refactor your code and remove unnecessary errors

**Pull request merged**